hadoop-hdds/docs/content/design/secure-s3.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Secure S3 keys management
+summary: Improving security regarding s3 keys management
+date: 2023-03-10
+jira: HDDS-8132
+status: implemented
+author: Maksim Myskov, Mikhail Pochatkin
+---
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+# Abstract
+
```

Comment:
_Add what is to be done:_
Extend Ozone S3 secret key management to support centralized remote S3 secret storage.  To add TTL to secrets.  To also implement S3 gateway endpoint to provide get s3 secret and secret rotataion.